src/Negotiation/Matched.php

```diff
@@ -2,7 +2,7 @@
 
 namespace Negotiation;
 
-final class Match
+final class Matched
```

Comment:
I'm not sure what the best name is to use for this class, so suggestions are welcome for an alternative class name

Reply:
Just simple `Match_` ?

Reply:
`AcceptMatch` maybe?